### PR TITLE
Loadouts #1 - Wastelander, DFS

### PR DESCRIPTION
**Pull request by AlcoholismIncarnate**
**WARNING: THIS PR WAS PUSHED **KNOWING OF MAJOR BUGS****

- Items will not spawn within the backpack for DF forces. **_Carl and myself are working to bugfix this_**
This was pushed on his request.

## About The Pull Request

- Adds loadouts kits to the WASTELANDER and DFS factions.
- All factions will get this treatment, once the bug is found and fixed.

**_DFS_**
- Militant in nature. They've access to raider-type gear for anybody that isn't just some grunt
- Agent did *not* get a loadout kit. 'nor will the advisor for this push. They are planned to get them.

**_GMB_**
- Vermonts own little green-wannabe-army-men
- Generally supplied with better equipment than the other factions. But suffer from a near total lack of armor for lesser troops.
  Because who would wanna cover up those neighborhood-watch uniforms?
## Why It's Good For The Game

Loadout kits get peoples little dopamine receptors going and increases general engagement. 
Plus Carl'll break my kneecaps if I don't help add them.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added new things
code: changed some code

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -6,4 +6,4 @@
 # Ranks can be anything defined in admin_ranks.txt                                            #
 # NOTE: if the rank-name cannot be found in admin_ranks.txt, they will not be adminned! ~Carn #
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
-###############################################################################################
+###############################################################################################
```
Comment (Cupax3) [inline config/admins.txt:9]: ?

File: code/modules/jobs/job_types/bl_fields.dm [hunk 2]
```diff
@@ -100,10 +130,41 @@
...
+// Mercenaries trade armor for a solid rifle. Something to show off for the Caravan.
+/datum/outfit/loadout/grunt/mercenary
+	name = "Mercenary"
+	r_hand = /obj/item/gun/ballistic/automatic/rangemaster
+	l_hand = /obj/item/ammo_box/magazine/m762 // 1 loaded, two empty standard magazines.
+	suit = /obj/item/clothing/suit/armored/f13/light/leathermk2
+	backpack_contents = list(/obj/item/ammo_box/magazine/m762/empty,
+							/obj/item/ammo_box/magazine/m762/empty,
+							/obj/item/melee/onehanded/knife/bayonet)
+// Sadists rush into combat with strong medium armor, but no firearm. Likely conscripted Raiders that proved their worth.
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_fields.dm]:
```suggestion
							/obj/item/melee/onehanded/knife/bayonet)
							
// Sadists rush into combat with strong medium armor, but no firearm. Likely conscripted Raiders that proved their worth.
```

File: code/modules/jobs/job_types/bl_fields.dm [hunk 3]
```diff
@@ -100,10 +130,41 @@
...
+/datum/outfit/loadout/grunt/sadist
+	name = "Sadist"
+	r_hand = /obj/item/twohanded/fireaxe/bmprsword
+	l_hand = /obj/item/melee/onehanded/knife/bowie
+	suit = /obj/item/clothing/suit/armored/f13/medium/combat/scrapcombat
+	head = /obj/item/clothing/head/helmet/f13/combat
+	backpack_contents = list()
+// Punishers use heavy armor and shotguns to become bulwarks on the defence.
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_fields.dm]:
```suggestion
	backpack_contents = list()
	
// Punishers use heavy armor and shotguns to become bulwarks on the defence.
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 4]
```diff
@@ -83,11 +83,35 @@
...
+	/datum/outfit/loadout/walker/heavy,)
+
 /datum/outfit/job/gmb/walker
 	name = "Walker"
 	jobtype = /datum/job/gmb/walker
 	id = /obj/item/card/id/gmb_commander_keys
 
+/*/datum/outfit/loadout/walker/veteran
+	name = "Veteran"
+	backpack_contents = list()*/ // WAITING FOR THE CARL TO GIVE IDEA
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm:98]: See House Rules channel

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 5]
```diff
@@ -114,7 +166,20 @@
...
+
+/datum/outfit/loadout/volunteer/conscript
+	name = "Conscript"
+	r_hand = /obj/item/gun/ballistic/revolver/piperifle // YOU'RE A CONSCRIPT. BE HAPPY YOU EVEN GET A GUN.
+	backpack_contents = list(/obj/item/melee/onehanded/knife/bayonet,)
+
+/datum/outfit/loadout/volunteer/watchmen
+	name = "Watchmen"
+	r_hand = /obj/item/gun/ballistic/revolver/piperifle // YOU'RE A CONSCRIPT. BE HAPPY YOU EVEN GET A GUN.
+	backpack_contents = list(/obj/item/melee/onehanded/knife/bayonet,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
	backpack_contents = list(/obj/item/melee/onehanded/knife/bayonet)
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 6]
```diff
@@ -114,7 +166,20 @@
...
+	/datum/outfit/loadout/volunteer/watchmen,)
 
 /datum/outfit/job/gmb/volunteer
 	name = "Volunteer"
 	jobtype = /datum/job/gmb/volunteer
+
+/datum/outfit/loadout/volunteer/conscript
+	name = "Conscript"
+	r_hand = /obj/item/gun/ballistic/revolver/piperifle // YOU'RE A CONSCRIPT. BE HAPPY YOU EVEN GET A GUN.
+	backpack_contents = list(/obj/item/melee/onehanded/knife/bayonet,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
	backpack_contents = list(/obj/item/melee/onehanded/knife/bayonet)
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 7]
```diff
@@ -114,7 +166,20 @@
 	outfit = /datum/outfit/job/gmb/volunteer
 	exp_requirements = 60
 	exp_type = EXP_TYPE_WASTELAND
+	loadout_options = list(
+	/datum/outfit/loadout/volunteer/conscript,
+	/datum/outfit/loadout/volunteer/watchmen,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
	/datum/outfit/loadout/volunteer/watchmen)
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 8]
```diff
@@ -99,10 +123,38 @@
...
+							/obj/item/melee/onehanded/knife/bayonet)
+
+/datum/outfit/loadout/militia/hunter
+	name = "Hunter"
+	r_hand = /obj/item/gun/ballistic/rifle/repeater/cowboy
+	l_hand = /obj/item/ammo_box/magazine/internal/shot/tube357 // four loaded magazines.
+	backpack_contents = list(/obj/item/ammo_box/magazine/internal/shot/tube357,
+							/obj/item/ammo_box/magazine/internal/shot/tube357,
+							/obj/item/ammo_box/magazine/internal/shot/tube357,
+							/obj/item/melee/onehanded/knife/hunting,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
							/obj/item/melee/onehanded/knife/hunting)
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 9]
```diff
@@ -83,11 +83,35 @@
...
+/*/datum/outfit/loadout/walker/veteran
+	name = "Veteran"
+	backpack_contents = list()*/ // WAITING FOR THE CARL TO GIVE IDEA
+
+/datum/outfit/loadout/walker/marksmen
+	name = "Marksmen"
+	r_hand = /obj/item/gun/ballistic/rifle/enfield
+	l_hand = /obj/item/gun/ballistic/automatic/pistol/beretta/automatic
+	suit = /obj/item/clothing/suit/armored/f13/light/leathermk2
+	backpack_contents = list(/obj/item/ammo_box/magazine/m9mmds,) // Please add .308 when able, KOARL
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
	backpack_contents = list(/obj/item/ammo_box/magazine/m9mmds) // Please add .308 when able, KOARL
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 10]
```diff
@@ -83,11 +83,35 @@
 	access = list(ACCESS_GMB, ACCESS_GMB_MOTORPOOL)
 	minimal_access = list(ACCESS_GMB, ACCESS_GMB_MOTORPOOL)
 
+	loadout_options = list(
+	//datum/outfit/loadout/walker/veteran,
+	/datum/outfit/loadout/walker/marksmen,
+	/datum/outfit/loadout/walker/heavy,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
	/datum/outfit/loadout/walker/heavy)
```

File: code/modules/jobs/job_types/bl_fields.dm [hunk 11]
```diff
@@ -68,12 +68,42 @@
...
+	suit = /obj/item/clothing/suit/armored/f13/heavy/sulphite
+	head = /obj/item/clothing/head/helmet/f13/sulphitehelm
+	backpack_contents = list()
+
+/datum/outfit/loadout/enforcer/enforcer
+	name = "Enforcer"
+	r_hand = /obj/item/gun/ballistic/shotgun/automatic/combat/auto5
+	suit = /obj/item/clothing/suit/armored/f13/medium/combat/mk2
+	head = /obj/item/clothing/head/helmet/f13/combat/mk2
+	backpack_contents = list(/obj/item/ammo_box/shotgun/buck,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_fields.dm]:
```suggestion
	backpack_contents = list(/obj/item/ammo_box/shotgun/buck)
```

File: code/modules/jobs/job_types/bl_fields.dm [hunk 12]
```diff
@@ -68,12 +68,42 @@
...
 
+// Fourty-five submachinegun. For when business decisions become executive! And a badass coat.
+/datum/outfit/loadout/enforcer/taskmaster
+	name = "Taskmaster"
+	r_hand = /obj/item/gun/ballistic/automatic/smg/tommygun
+	l_hand = /obj/item/ammo_box/magazine/tommygunm45 // Fifty rounds on tap. Two spare sticks.
+	gloves = /obj/item/clothing/gloves/f13/military
+	suit = /obj/item/clothing/suit/armored/f13/medium/armoredcoat
+	backpack_contents = list(/obj/item/ammo_box/magazine/tommygunm45/stick/empty,
+							/obj/item/ammo_box/magazine/tommygunm45/stick/empty,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_fields.dm]:
```suggestion
							/obj/item/ammo_box/magazine/tommygunm45/stick/empty)
```

File: code/modules/jobs/job_types/bl_fields.dm [hunk 13]
```diff
@@ -68,12 +68,42 @@
 	access = list(ACCESS_DFS, ACCESS_DFS_BOSS)
 	minimal_access = list(ACCESS_DFS, ACCESS_DFS_BOSS)
 
+	loadout_options = list(
+	/datum/outfit/loadout/enforcer/taskmaster,
+	/datum/outfit/loadout/enforcer/peacemaker,
+	/datum/outfit/loadout/enforcer/enforcer,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_fields.dm]:
```suggestion
	/datum/outfit/loadout/enforcer/enforcer)
```

File: code/modules/jobs/job_types/bl_mountain.dm [hunk 14]
```diff
@@ -99,10 +123,38 @@
 	outfit = /datum/outfit/job/gmb/militia
 	exp_requirements = 120
 
+	loadout_options = list(
+	/datum/outfit/loadout/militia/hunter,
+	/datum/outfit/loadout/militia/musketeer,
+	/datum/outfit/loadout/militia/rifleman,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/bl_mountain.dm]:
```suggestion
	/datum/outfit/loadout/militia/rifleman)
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 15]
```diff
@@ -80,26 +80,30 @@
...
 	shoes = /obj/item/clothing/shoes/f13/military
 	r_hand = /obj/item/storage/backpack/duffelbag/scavengers
 	l_hand = /obj/item/pickaxe/drill
 	belt = /obj/item/storage/belt
 	backpack_contents = list(/obj/item/mining_scanner=1,
 							/obj/item/metaldetector=1,
 							/obj/item/shovel=1,
-							/obj/item/book/granter/trait/selection =1,)
+							/obj/item/book/granter/trait/techno =1,
+							/obj/item/storage/box/vendingmachine=1,)
```
Comment (Cupax3) [inline code/modules/jobs/job_types/wasteland.dm:104]: pick one style and stick with it, also comma at the end of list still present

File: code/modules/jobs/job_types/wasteland.dm [hunk 16]
```diff
@@ -80,26 +80,30 @@
...
-	head = /obj/item/clothing/head/welding
-	r_hand = /obj/item/weldingtool/largetank
-	backpack_contents = list(/obj/item/book/granter/trait/selection =1)
-
-/datum/outfit/loadout/scavenger
-	name = "Scavenger"
+	r_hand = /obj/item/gun/ballistic/revolver/piperifle
+	gloves = /obj/item/clothing/gloves/f13/leather/fingerless
+	backpack_contents = list()
+
```
Comment (Cupax3) [inline code/modules/jobs/job_types/wasteland.dm:92]: mix of tabs and whitespace? bad

File: code/modules/jobs/job_types/wasteland.dm [hunk 17]
```diff
@@ -109,16 +113,13 @@
...
 		/obj/item/cultivator = 1,
 		/obj/item/reagent_containers/glass/bucket = 1,
-		/obj/item/storage/bag/plants/portaseeder = 1,
-		/obj/item/book/granter/trait/selection =1,
-		)
-
-/datum/outfit/loadout/medic
-	name = "Wasteland Doctor"
+		/obj/item/storage/bag/plants/portaseeder = 1,)
+// Bonecutter - Bonecutter don't exactly have the best medical skills in the wasteland. But they're still invaluable compared to most! For medically focused players.
```
Comment (Cupax3) [inline code/modules/jobs/job_types/wasteland.dm]:
```suggestion
		/obj/item/storage/bag/plants/portaseeder = 1)
		
// Bonecutter - Bonecutter don't exactly have the best medical skills in the wasteland. But they're still invaluable compared to most! For medically focused players.
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 18]
```diff
@@ -133,9 +134,15 @@
 							/obj/item/folder=1,
 							/obj/item/pen/fountain=1,
 							/obj/item/storage/firstaid/ancient=1,
-							/obj/item/book/granter/trait/selection =1,
-		)
-
+							/obj/item/book/granter/trait/lowsurgery =1,)
+// Chemrunner - Chemrunners aren't exactly the most reliable folks in the wasteland, but they can find a strong niche.
```
Comment (Cupax3) [inline code/modules/jobs/job_types/wasteland.dm]:
```suggestion
							/obj/item/book/granter/trait/lowsurgery =1)
							
// Chemrunner - Chemrunners aren't exactly the most reliable folks in the wasteland, but they can find a strong niche.
```

**Review by Cupax3 — CHANGES_REQUESTED** [review]

File: config/admins.txt (hunk 1 above)
Comment (AlcoholismIncarnate) [reply config/admins.txt:9]: This was how the PR saw me removing myself as host. Don't ask. I did a brainfart.

**Pull request closed without merging**